packages/edit-post/src/components/header/mode-switcher/index.js

```diff
@@ -55,8 +55,24 @@ function ModeSwitcher() {
 		return null;
 	}
 
-	if ( ! isRichEditingEnabled || ! isCodeEditingEnabled ) {
-		return null;
+	if ( ! isRichEditingEnabled ) {
+		return (
+			<MenuGroup label={ __( 'Editor' ) }>
+				<Notice isDismissible={ false }>
```

Comment:
I'm assuming `<Notice>` isn't correct in this context. What would be a better component? And is this an appropriate use of `<MenuGroup>`?

Reply:
Yes you are right. We should keep the menu group and alter `choices` for this situation. See the `MenuItemsChoice` component for options on choices.